README.md

```diff
@@ -33,7 +33,7 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖  Bookstore: add reducers and actions <a name="about-project"></a>
+# 📖  Bookstore: styling <a name="about-project"></a>
```

Comment:
- [ ] Please provide the details about your project ie: what you did in the project so that any user visiting your app can know details about the project also it makes your README professional.